### PR TITLE
Buffs the Giga Drill

### PR DESCRIPTION
**Pull request by icantthinkofanameritenow**
Addresses https://github.com/d3athrow/vgstation13/issues/8412 by making the drill faster and able to drill through multiple tiles at once.

### CODE REVIEW

**Comment by Exxion** [comment]
Are you sure you did the multiple tiles thing correctly? The tabbing doesn't look right and I'm... not really sure what an extra `if` would do anyway.

**Comment by PJB3005** [comment]
Just remove the drill time entirely. 

Even with it removed it's slower than a diamond drill.

File: code/modules/research/xenoarchaeology/artifact/artifact_gigadrill.dm [hunk 1]
```diff
@@ -29,6 +29,7 @@
 			anchored = 1
 			spawn(drill_time)
 				if(get_turf(src) == drilling_turf && active)
+				if(get_dir(src,M)&src.dir)
```
Comment (PJB3005) [inline code/modules/research/xenoarchaeology/artifact/artifact_gigadrill.dm:31]: Er....

**Comment by PJB3005** [comment]
This isn't gonna do shit even if it did compile.

File: code/modules/research/xenoarchaeology/artifact/artifact_gigadrill.dm [hunk 2]
```diff
@@ -5,7 +5,6 @@
 	icon = 'icons/obj/mining.dmi'
 	icon_state = "gigadrill"
 	var/active = 0
-	var/drill_time = 10
```
Comment (PJB3005) [inline code/modules/research/xenoarchaeology/artifact/artifact_gigadrill.dm:8]: So you're just gonna remove the variable but not remove any of the references to it?

You're not even trying are you?

**Comment by icantthinkofanameritenow** [comment]
@PJB3005 
Well, fuck. So much for this then. 

I think I'll take a break from coding until I have a better idea of how the fuck I'm supposed to do it.

**Comment by Exxion** [comment]
Don't worry, this is probably more in-depth a fix than a new coder should try
I started on something too advanced and my code was so bad I'll probably have to mostly rewrite spacepod equipment now

**Pull request closed without merging**